packages/ui-kit/src/components/SimpleFilter/SimpleFilter.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { SyntheticEvent, useEffect, useRef } from 'react'
+import { useTopValues } from '../../hooks'
+import { FilterInput, FilterOperator, getTimeZone } from '../../helpers'
+
+import { Autocomplete } from '../Autocomplete'
+import { AutocompleteOption } from '../Autocomplete/Autocomplete.types'
+import { useFilters } from '../FilterProvider/useFilters'
+
+import { SimpleFilterProps } from './SimpleFilter.types'
+import { withContainer } from '../withContainer'
+import { ErrorFallback } from '../ErrorFallback'
+import { Loader } from '../Loader'
+import componentStyles from './SimpleFilter.module.scss'
+import { useLog } from '../Log'
+
+const SimpleFilterComponent = ({
+  autocompleteProps,
+  columnName: columnNameProp = '',
+  query,
+  error,
+  loading,
+  loaderProps,
+  options = []
+}: SimpleFilterProps) => {
+  const id = useRef(Symbol()).current
+
+  const isStatic = !query
+
+  const { filters, setFilters } = useFilters()
+
+  const columnName = query?.columnName ?? columnNameProp
+  const zone = query?.timeZone ?? getTimeZone()
+
+  const log = useLog()
+
+  const { data, error: queryError, isLoading } = useTopValues({ ...query, timeZone: zone })
+
+  const isError = queryError != null || error != null
+
+  const handleChange = (_: SyntheticEvent<Element, Event>, selectedOption: AutocompleteOption | string | null) => {
+    if (selectedOption == null) return
+
+    const newFilter: FilterInput = {
```

Comment:
I personally think that as much as we should avoid naming var/const with 'new' such newFilter, newList. 

Instead of this we should give more definetive naming inputFilter, filterList.

wdyt?